prow/config/config.yaml

```diff
@@ -1145,6 +1145,144 @@ postsubmits:
                 path: config.json
 
 
+  - name: build-push-image-argocd-kpt-repo-plugin-release-conf
+    cluster: default
+    always_run: true
+    branches:
+    - "^v.*$"
+    annotations:
+      description: Build and Push ArgoCD KPT Repo Plugin Image to DockerHub when release tag is created
+    decorate: true
+    decoration_config:
+      censor_secrets: true
+    max_concurrency: 0
+    spec:
+      containers:
+      - name: kaniko
+        image: gcr.io/kaniko-project/executor:debug
+        command:
+        - "/bin/sh"
+        - "-c"
+        - |
+          executor --context=${PWD}/gitops-tools/kpt-argocd-cmp/kpt-repo/ --dockerfile=Dockerfile --destination=nephio/kpt-repo-argo-cmp:${PULL_BASE_REF}
```

Comment:
@efiacor so the images in this case will be named `kpt-repo-argo-cmp` and `kpt-render-argo-cmp`? I believe the current patch assumes `kpt-repo-plugin` and `kpt-render-plugin` so we'll need to conform one to the other

Reply:
(no preference either way, if kpt-...-argo-cmp is more explicit that's fine

Reply:
That's a good question, how are other images usually referenced? We can change that and the name in a single PR to fix the patch once this is merged/decided